perllib/FixMyStreet/Roles/SCP.pm

```diff
@@ -27,6 +25,14 @@ sub waste_cc_get_redirect_url {
     $p->set_extra_metadata('redirect_id', $redirect_id);
     $p->update;
 
+    my $backUrl;
+    if ($back eq 'bulky') {
+        # Need to pass through property ID as not sure how to work it out once we're back
+        $backUrl = $c->uri_for('pay_cancel', $p->id, $redirect_id ) . '?property_id=' . $c->stash->{property}{id};
```

Comment:
SCP wants the param to be URL encoded otherwise it complains about an invalid request. Guess the space in the postcode is throwing it off.